tests/unit/test_charm.py

```diff
@@ -124,41 +120,6 @@ def test_with_ingress_relation(self, harness):
 
         assert harness.charm.ingress_relation.status == ActiveStatus()
 
-    def test_warning_if_ingress_missing(self, harness, mocker):
```

Comment:
suggestion (non-blocking): shall we replace this with a similar test that ensures the new conflict detector works as expected?